roles/centos7/dashboards/tasks/main.yml

```diff
@@ -3,6 +3,12 @@
 - hostname:
     name: "{{ inventory_hostname }}"
 
+- name: Install the libselinux-python for Amazon Linux 2
```

Comment:
What is this pkg used for?

Reply:
next step, we are disabling selinux and this package is required for that. RHEL7/centos7 having this package by default .

Reply:
Got it.

Reply:
Just a note: can you also add a comment on why selinux needs to be disabled?
Not requirement but would be nice if other users are concerning.

Thanks.

Reply:
selinux is used for additional security measures that requires lot of configurations. Not many using that nowadays. Moreover its already disabled by default in Amazon Linux 2 and I am just ensuring its stays disabled.

Reply:
Got it.